files/image_config/caclmgrd/caclmgrd

```diff
@@ -204,6 +204,19 @@ class ControlPlaneAclManager(object):
                                   ("DST_IP" in rule_props and rule_props["DST_IP"])):
                                 table_ip_version = 4
 
+                        if ((("SRC_IPV6" in rule_props and rule_props["SRC_IPV6"]) or
```

Comment:
Suggest to create utility functions is_v4_rule(rule_props), is_v6_rule(rule_props) to make this condition more readable

Reply:
addressed in the next commit.